frontend/services/entities/hosts.ts

```diff
@@ -602,22 +602,16 @@ export default {
   },
 
   getHostCertificates: (
-    hostId: number
+    hostId: number,
+    page = 0,
+    perPage = 10
```

Comment:
This should also include sort params.

Reply:
the sort is meant to be client side.

Reply:
I think the Figma wasn't updated, but this was changed when the story was estimated. Here's the API [specs](https://github.com/fleetdm/fleet/pull/26457). I'll plan on incorporating this change as I do my QA review of the story.